dex/testing/eth/harness.sh

```diff
@@ -184,9 +185,9 @@ echo "Sending 5000 eth to delta and gamma."
 "${NODES_ROOT}/harness-ctl/alpha" "attach --preload ${NODES_ROOT}/harness-ctl/send.js --exec send(\"${ALPHA_ADDRESS}\",\"${GAMMA_ADDRESS}\",${SEND_AMT})"
 "${NODES_ROOT}/harness-ctl/alpha" "attach --preload ${NODES_ROOT}/harness-ctl/send.js --exec send(\"${ALPHA_ADDRESS}\",\"${DELTA_ADDRESS}\",${SEND_AMT})"
 
-# Our transactions will PROBABLY appear in one of these blocks.
 echo "Mining some blocks"
-"${NODES_ROOT}/harness-ctl/mine-beta" "15"
+"${NODES_ROOT}/harness-ctl/mine-alpha" "2"
+"${NODES_ROOT}/harness-ctl/mine-beta" "2"
```

Comment:
So this ended up working better?  It wasn't in the previous revision.

Reply:
Yes. It appears that alternating is also important. Although, it was alternating at 15 and 15 blocks, it seems that is overkill, and it also sometimes cause the second node mined to fail to accept blocks from the first node, apparently because there were too many blocks from the first node initially.